src/native/libs/System.Globalization.Native/pal_locale.m

```diff
@@ -28,6 +30,129 @@
+            break;
+        case LocaleString_DecimalSeparator:
+            value = [currentLocale.decimalSeparator UTF8String];
+            // or value = [[currentLocale objectForKey:NSLocaleDecimalSeparator] UTF8String];
+            break;
+        case LocaleString_MonetarySymbol:
+            value = [currentLocale.currencySymbol UTF8String];
+            break;
+        case LocaleString_Iso4217MonetarySymbol:
+            // check if this is correct, check currencyISOCode
+            value = [currentLocale.currencySymbol UTF8String];
+            break;
+        case LocaleString_CurrencyEnglishName:
+            value = [[gbLocale localizedStringForCurrencyCode:currentLocale.currencyCode] UTF8String];
+            break;
+        case LocaleString_CurrencyNativeName:
+            value = [[currentLocale localizedStringForCurrencyCode:currentLocale.currencyCode] UTF8String];
+            break;
+        case LocaleString_AMDesignator:
+            value = [dateFormatter.AMSymbol UTF8String];
+            break;
+        case LocaleString_PMDesignator:
+            value = [dateFormatter.PMSymbol UTF8String];
+            break;
+        case LocaleString_PositiveSign:
+            value = [numberFormatter.plusSign UTF8String];
+            break;
+        case LocaleString_NegativeSign:
+            value = [numberFormatter.minusSign UTF8String];
+            break;
+        case LocaleString_Iso639LanguageTwoLetterName:
+            // check if this is correct
+            value = [[currentLocale objectForKey:NSLocaleLanguageCode] UTF8String];
+            break;
+        case LocaleString_Iso3166CountryName:
+            value = [currentLocale.countryCode UTF8String];
+            break;
+        case LocaleString_NaNSymbol:
+            value = [numberFormatter.notANumberSymbol UTF8String];
+            break;
+        case LocaleString_PositiveInfinitySymbol:
+            value = [numberFormatter.positiveInfinitySymbol UTF8String];
+            break;
+        case LocaleString_NegativeInfinitySymbol:
+            value = [numberFormatter.negativeInfinitySymbol UTF8String];
+            break;
+        case LocaleString_PercentSymbol:
+            value = [numberFormatter.percentSymbol UTF8String];
+            break;
+        case LocaleString_PerMilleSymbol:
+            value = [numberFormatter.perMillSymbol UTF8String];
+            break;
+        // TODO find mapping for below cases
+        // https://github.com/dotnet/runtime/issues/83514
+        case LocaleString_Digits:
+        case LocaleString_MonetaryDecimalSeparator:
+        case LocaleString_MonetaryThousandSeparator:
+        case LocaleString_Iso639LanguageThreeLetterName:
+        case LocaleString_ParentName:
+        case LocaleString_Iso3166CountryName2:            
+        default:
+            value = "";
+            break;
+    }
+    NSLog(@"Globalization GlobalizationNative_GetLocaleInfoStringNative value: %s", value);
```

Comment:
```suggestion
```
This logging should be deleted once the change is finalized.

Reply:
Deleted.